app/scripts/modules/core/instance/instanceType.service.ts

```diff
@@ -85,7 +86,7 @@ export class InstanceTypeService {
 
   private getInstanceTypeCategory(cloudProvider: string, instanceType: string): ng.IPromise<IInstanceTypeCategory> {
     return this.getCategories(cloudProvider).then((categories: IInstanceTypeCategory[]) => {
-      return (categories || []).find(c => c.families.some(f => f.instanceTypes.some(i => i.name === instanceType)));
+      return (categories || []).find(c => c.families.some(f => f.instanceTypes.some(i => i.name === instanceType || (i.nameRegex && i.nameRegex.test(instanceType)))));
```

Comment:
I dunno, this makes sense to me. It doesn't affect other cloud providers, and provides a hook for future cloud providers to customize the name matching.